test/api/requests/build.test.js

```diff
@@ -530,34 +535,85 @@ describe('Build API', () => {
     });
 
 
-    it('should report the build\'s status back to github', (done) => {
+    it('should report the build\'s status back to github', async () => {
+      nock.cleanAll();
```

Comment:
maybe we should just do this in the `beforeEach` instead of repeating it in every test